api/PerformanceResourceTiming.json

```
@@ -732,7 +732,7 @@
           "spec_url": "https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-tojson",
           "support": {
             "chrome": {
-              "version_added": "43"
+              "version_added": "45"
```

https://bugzilla.mozilla.org/show_bug.cgi?id=1047848 implemented JSON behavior apparently. How did the collector come to version 45?

```suggestion
              "version_added": "34"
```

---

That's a Firefox bug, but this is a PR that updates Chrome!

---

hahaha I guess I should take a review break!